docs/pages/enroll-resources/database-access/enroll-aws-databases/rds.mdx

```diff
@@ -295,6 +295,8 @@ Retrieve credentials for the database and connect to it as the `alice` user:
 $ tsh db connect --db-user=alice --db-name=postgres rds-example
 ```
 
+(!docs/pages/includes/database-access/pg-access-webui.mdx!)
```

Comment:
not sure this is the right place to inject the tip (like right in the middle of `tsh` instructions). Maybe we can move it at the end of the section and drop the `tip`? Or move it beginning of this section? Or show web UI instructions in a tab?

Reply:
I've considered this, but since we're introducing it as an alternative to the `tsh db connect` access, users will likely first try out their databases the current way (using `tsh`) and then explore using it with the Web UI. It also creates one less branch for them to decide on.